File: julia/leap/leap.jl
"""
    is_leap_year(year)

Return `true` if `year` is a leap year in the gregorian calendar.

on every year that is evenly divisible by 4
  except every year that is evenly divisible by 100
    unless the year is also evenly divisible by 400
"""
function is_leap_year(year)
    if mod(year, 4) == 0
        if mod(year, 100) == 0
            if mod(year, 400) == 0
                return true
            else
                return false
            end
        else
            return true
        end
    else
        return false
    end
end

File: python/leap/leap.py
"""
dETermine the if the year is a leap year
"""


def leap_year(year):
    """
    :param year: int Year to test
    :return: bool
    """

    if ((year % 4 == 0) and (year % 100 != 0)) or (year % 400 == 0):
        return True

    return False
